app/queue.py

```diff
@@ -107,9 +107,7 @@ def __init__(self, suffix=None, expire_inflight_after_seconds=300) -> None:
 
         # Create async event loop for CloudWatch metrics
         try:
-            loop = (
-                asyncio.get_event_loop()
-            )  # This will fail to create a new event loop if called outside the main thread. https://bugs.python.org/issue39381
+            asyncio.get_event_loop()  # This will fail to create a new event loop if called outside the main thread. https://bugs.python.org/issue39381
```

Comment:
Hmm I wonder why there was an assignment. Nothing to do with the following one in the except block?

Reply:
this is just to test that an event loop exists. If it fails it'll create one, so i had an unnecessary assignment when i introduced this the first time.